README.md

```diff
@@ -1,50 +1,81 @@
 # cargo-instruments
 
-Easily generate [Instruments] traces for your rust crate.
+Easily profile your rust crate with Xcode [Instruments].
 
-`cargo-instruments` is the glue between cargo and Xcode's bundled profiling
+`cargo-instruments` is the glue between Cargo and Xcode's bundled profiling
 suite. It allows you to easily profile any binary in your crate, generating
 files that can be viewed in the Instruments app.
 
 ![Instruments Time Profiler](https://raw.githubusercontent.com/cmyr/cargo-instruments/screenshots/instruments_time1.png)
 ![Instruments System Trace](https://raw.githubusercontent.com/cmyr/cargo-instruments/screenshots/instruments_sys1.png)
 
+This is a proposal refactoring of [Colin Rofls' excellent
+work](https://github.com/cmyr/cargo-instruments).
```

Comment:
We won't need this comment when we merge 😉

Reply:
Absolutely